### PR TITLE
src: fix up cpplint errors in node_jsvmapi.h

### PR DESCRIPTION
**Pull request by mhdawson**
Starting to work down the list of errors we need correct
before submitting PR to core

### CODE REVIEW

**Comment by mhdawson** [comment]
CI run: https://ci.nodejs.org/view/x%20-%20Abi%20stable%20module%20API/job/node-test-commit-linux-abi/

File: src/node_jsvmapi.h [hunk 1]
```diff
@@ -203,42 +270,54 @@ NODE_EXTERN napi_status napi_make_callback(napi_env e,
...
+                                                napi_callback_info cbinfo,
+                                                int* result);
+NODE_EXTERN napi_status napi_get_cb_args(napi_env e,
+                                         napi_callback_info cbinfo,
+                                         napi_value* buffer,
+                                         int bufferlength);
+NODE_EXTERN napi_status napi_get_cb_this(
+                                napi_env e,
+                                napi_callback_info cbinfo,
+                                napi_value* result);
```
Comment (jasongin) [inline src/node_jsvmapi.h]: This one is incorrectly aligned.

**Review by jasongin — APPROVED** [review]

File: src/node_jsvmapi.h (hunk 1 above)
Comment (mhdawson) [reply src/node_jsvmapi.h]: I'll double check but I had to do that in several cases to avoid hitting the 80 char limit

Comment (mhdawson) [reply src/node_jsvmapi.h]: Looks like that one can be aligned, fixing.

**Comment by mhdawson** [comment]
Pushed commit to address comments so far.  Will squash when landing.

**Comment by mhdawson** [comment]
Landed as 47b3ca0223c691273bd4a3614a3c9a20de7b8d2d

**Pull request closed without merging**